src/SourceLocator/Reflection/SourceStubber.php

```diff
@@ -106,10 +115,7 @@ private function addDocComment(BuilderAbstract $node, CoreReflector $reflection)
         }
     }
 
-    /**
-     * @param Class_|Interface_|Trait_ $classNode
-     */
-    private function addClassModifiers(Declaration $classNode, CoreReflectionClass $classReflection) : void
+    private function addClassModifiers(Class_ $classNode, CoreReflectionClass $classReflection) : void
```

Comment:
Is `ClassLike` not more appropriate here? As this would no longer accept `Interface_` or `Trait_` I think, which I believe should be accepted so we can make them `abstract` (unless I'm mistaken)

Reply:
Traits/Interfaces can't be `abstract` though? Or `final`?

Reply:
No fix needed